meson_options.txt

```diff
@@ -252,5 +252,7 @@ option('zshcompletiondir', type : 'string',
 
 option('tests', type : 'combo', choices : ['true', 'unsafe'],
        description : 'enable extra tests with =unsafe')
+option('slow-tests', type : 'boolean', value : 'false',
+       description : 'run the slow tests by default')
```

Comment:
This is bikeshedding matter of course, but shouldn't we keep the slow tests on by default, and only provide that option to turn them off? (i. e. `value: 'true'`)